### PR TITLE
Added documentation and example for :class:`.CyclicReplace`

### PR DESCRIPTION
**Pull request by jeertmans**
## Overview: What does this pull request change?

This PR adds a documentation to the `CyclicReplace` transformation, with an example.

## Motivation and Explanation: Why and how do your changes improve the library?

I think keeping the documentation up-to-date is important.

## Links to added or changed documentation pages

https://manimce--2978.org.readthedocs.build/en/2978/reference/manim.animation.transform.CyclicReplace.html

## Reviewer Checklist
- [x] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [x] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [x] If applicable: newly added functions and classes are tested

### CODE REVIEW

**Comment by alembcke** [comment]
Thank you for contributing this, I agree that keeping the documentation up-to-date is important.

What are your thoughts about repeating the `CyclicReplace` four times, so that the video plays in a perfect loop?

**Comment by jeertmans** [comment]
Indeed, @alembcke, that's a good idea! The animation now looks much nicer ;-)

File: manim/animation/transform.py [hunk 1]
```diff
@@ -691,6 +691,40 @@ def _init_path_func(self) -> None:
...
+    ----------
+    mobjects
+        :class:`.Mobject`
+        List of Mobject to be transformed.
+    path_arc
+        The arc angle (in radians) that the points of ``mobject`` will follow to reach
+        the points of the target if using a circular path arc, see ``path_arc_centers``.
+        See also :func:`manim.utils.paths.path_along_arc`.
+    kwargs
+        Further keyword arguments that are passed to :class:`Transform`.
```
Comment (behackl) [inline manim/animation/transform.py:711]:
```suggestion
        Further keyword arguments that are passed to :class:`.Transform`.
```

File: manim/animation/transform.py [hunk 2]
```diff
@@ -691,6 +691,40 @@ def _init_path_func(self) -> None:
...
+
+        class CyclicReplaceExample(Scene):
+            def construct(self):
+
+                group = VGroup(Square(), Circle(), Triangle(), Star()).arrange(RIGHT)
+
+                self.add(group)
+
+                for i in range(4):
+                    self.play(CyclicReplace(*group))
```
Comment (behackl) [inline manim/animation/transform.py:725]: Minor changes to improve readability.

```suggestion
        class CyclicReplaceExample(Scene):
            def construct(self):
                group = VGroup(Square(), Circle(), Triangle(), Star())
                group.arrange(RIGHT)
                self.add(group)

                for _ in range(4):
                    self.play(CyclicReplace(*group))
```

**Review by behackl — CHANGES_REQUESTED** [review]
Thank you very much for your contribution! In general this is fine, I just left a few suggestions for improvement. Please let me know whether you are able to take a look and apply them / discuss them -- and otherwise we will apply them in a while and then merge this.

**Comment by jeertmans** [comment]
Hi @behackl, Thanks for the review! I am perfectly fine with all suggested changes! It’s my first contribution to Manim, so I expected to not exactly match your common style guidelines :)

**Review by behackl — APPROVED** [review]
Thank you!

**Pull request merged**